packages/block-library/src/embed/util.js

```diff
@@ -182,12 +182,16 @@ export const removeAspectRatioClasses = ( existingClassNames ) => {
 	}
 	const aspectRatioClassNames = ASPECT_RATIOS.reduce(
 		( accumulator, { className } ) => {
-			accumulator[ className ] = false;
+			accumulator.push( className );
 			return accumulator;
 		},
-		{ 'wp-has-aspect-ratio': false }
+		[ 'wp-has-aspect-ratio' ]
 	);
-	return classnames( existingClassNames, aspectRatioClassNames );
+	let outputClassNames = existingClassNames;
+	for ( const className of aspectRatioClassNames ) {
+		outputClassNames = outputClassNames.replace( className, '' );
+	}
+	return outputClassNames.trim();
```

Comment:
Can't we do it all in the reduce call? Like, start with `existingClassNames` and do a `replace` call on every aspect ratio class name right there? What way we won't need to perform a separate manipulation after the reduce call.

Reply:
Happy to do it that way in a follow-up. The reason I did it this way is because I wanted to only replace the classnames logic specifically, and try not to touch pre-existing code much. I agree that now that classnames/dedupe is not being used there's a more efficient way to write this 👍

Reply:
Sure. A follow-up sounds good 👍